### DISCUSSION TITLE
Suggest nice & simple design ideas for a minimalist landing page

### DISCUSSION THREAD

**Discussion by sidharthgehlot** [developersIndia/developersIndia#6] [Show and tell]
We've finally decided to kickoff setting up & developing the official website for [developersIndia](https://reddit.com/r/developersindia) community! So, if you've any ideas or some designing skills, please comment down below with an example. We would appreciate your contributions to the project & any sort of help are welcomed.

That said, for more info on what kind of landing are we looking for, refer to #20

**Comment by jarmos-san** [comment]
Thanks to @OmkarK45 & @sushilburagute we've an amazing design template! You can take a look at it in this [Figma template](https://www.figma.com/file/jxpIq8GzilqzgyLpFBySP2/Developers-India-%3C3?node-id=0%3A1). We'll be using it as a source of inspiration to build the first iteration of the official landing page.

For those of you wondering here are some examples of what the hero section of the template looks like:

![Hero](https://user-images.githubusercontent.com/31373860/152386922-725be6b9-eadf-48bb-9303-b172a7d2d5ec.png)
![AlternativeHomePage](https://user-images.githubusercontent.com/31373860/152386934-e732eef8-7411-4380-a50b-0a70ebe75332.png)

If you've design skills & would like to contribute to some FOSS projects, we appreciate your contributions as well! Please feel free to share example of what your interpretation of the website would look like in the comment section below.
